Thorium-Sim/thorium-nova
# Station Layout

## Description


Adds a station layout with the ability to apply the theme to it. Adds a few cards, as well as effects.

Sorry, it's a monster.

## Related Issue








Closes #134 
Closes #135 

## How do you know the changes work correctly?






Manual Test
- Make sure the default plugin is up to date (check the readme)
- Start a flight
- Assign client to station
- The default theme should be visible.

## Screenshots (if appropriate):

## Types of changes



      change)

## Checklist:
